sites/smart-cities/resources/convert.js

```diff
@@ -1,49 +0,0 @@
-#!/usr/bin/env node -r @std/esm
```

Comment:
Why do these get deleted? They shouldn't be. `npm run sc:generate-city-data` needs to still convert the spreadsheet.

Reply:
This is a great question indeed, let me look into it!